### DISCUSSION TITLE
Part 4

### DISCUSSION THREAD

**Discussion by LayZeeDK** [this-is-learning/rxjs-fundamentals-course#4] [Content contributions]
Discussing and coordinating [Part 4](https://github.com/this-is-learning/rxjs-fundamentals-course/blob/main/part-4/part-4.md) efforts.

**Comment by LayZeeDK** [comment]
# Contributors
We're looking for contributors to actively work on this part 👐

**Reply by kosich** [reply]
The current title "How RxJS is used by Angular" seems to promise an expert level review of Angular internals.
I'd suggest a slight change that might make it more appealing to newcomers: something like "How to use RxJS with Angular"
What do you think?

**Reply by LayZeeDK** [reply]
That makes sense, @kosich.
